src/SDKs/DataFactory/DataFactory.Tests/JsonSamples/LinkedServiceJsonSamples.cs

```diff
@@ -28,6 +28,31 @@ public class LinkedServiceJsonSamples : JsonSampleCollection<LinkedServiceJsonSa
     }
 }";
 
+        [JsonSample]
```

Comment:
nit: please fix indentation

Reply:
This is already formatted. And the indentation is following other examples.